### PR TITLE
Update workers changelog and cache API limits.

### PR DESCRIPTION
**Pull request by jp4a50**

### CODE REVIEW

File: content/workers/platform/limits.md [hunk 1]
```diff
@@ -96,14 +96,11 @@ On the Unbound billing model, scheduled Workers ([Cron Triggers](/workers/config
 | ----------------------------- | ------------- | ------- | ------- | ------- |
 | [Max object size](#cache-api-limits) | 512 MB | 512 MB  | 512 MB  | 512 MB  |
 | [Calls/request](#cache-api-limits)   | 50     | 50      | 1,000   | 1,000   |
-| [Storage/request](#cache-api-limits) | 5 GB   | 5 GB    | 5 GB    | 5 GB    |
```
Comment (jp4a50) [inline content/workers/platform/limits.md]: @fhanau I inferred that these changes should be made from your release note and also reworded the release note a little. PTAL.

File: data/changelogs/workers.yaml [hunk 2]
```diff
@@ -2,6 +2,11 @@
 link: "/workers/platform/changelog/"
 productName: Workers
 entries:
+- publish_date: '2024-04-03'
+  description: |-
+    - There is no longer an explicit limit on the total amount of data which may be uploaded with Cache API [`put()`](/workers/runtime-apis/cache/#put) per request. Other [Cache API Limits](/workers/platform/limits/#cache-api-limits) continue to apply.
+    - The Web standard `ReadableStream.from()` API is now implemented. The API enables creating a `ReadableStream` from a either a sync or async iterable.
+    - When the `brotli_content_encoding` compatibility flag is enabled, Workers will now support the `br` Content-Encoding and can request and respond with data encoded using the brotli compression algorithm.
```
Comment (jp4a50) [inline data/changelogs/workers.yaml]: Looks like I need to add a docs entry for this compatibility flag like Sam did in this PR: https://github.com/cloudflare/cloudflare-docs/pull/13725

Alternatively @fhanau could look at doing it.

- [x] Do it

**Comment by fhanau** [comment]
I made a docs PR for this earlier #13581

File: data/changelogs/workers.yaml (hunk 2 above)
Comment (fhanau) [reply data/changelogs/workers.yaml]: Describing brotli support accurately has been a challenge, but I have added documentation for it in #13684. I had to look into how the runtime behaves in one edge case but that's done now, should be able to finalize and merge this today.

Comment (irvinebroque) [reply data/changelogs/workers.yaml]: Nice — those docs are great

Comment (irvinebroque) [reply data/changelogs/workers.yaml]: Ship it!

**Review by irvinebroque — APPROVED** [review]

**Comment by fhanau** [comment]
Thank you, LGTM!

**Review by fhanau — APPROVED** [review]

**Pull request merged**